pkg/volume/local/local.go

```diff
@@ -208,6 +207,28 @@ func (m *localVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 		return nil
 	}
 
+        oldMount, err := mount.IsAlreadyMounted(dir)
```

Comment:
I think you need to use m.globalPath and not dir.  dir is the pod's mount directory, which you won't find more than once.